src/pages/Options/index.css

```diff
@@ -22,7 +23,8 @@ header h2 {
 }
 
 #app-container {
-  height: 100vh;
+  height: 100%;
+  min-height: 100vh;
```

Comment:
did you check that this doesn't break the other screens which open up in via options.html?
e.g. any of the collectibles screens when you click 'Open gallery' button, or the ledger send flows

Reply:
Yes, I did! Attached screenshots to a new comment.